specs/switch.md

```diff
@@ -42,8 +42,8 @@ Typically it's best to include a checked and unchecked message but it is not req
 
 *Slots:*
 - `default`
-- `unchecked-message`
-- `checked-message`
+- `unchecked-message` pass "content" class for default styling
+- `checked-message` pass "content" class for default styling
```

Comment:
I mentioned above, but I think we should just apply this for whatever gets passed here.